source/configuration_manual/fts/dovecot.rst

```diff
@@ -232,14 +232,12 @@ Example configuration using OBOX::
     fts_dovecot = yes
   }
 
-  fts_driver = dovecot
-
-  # Fall back to built in search.
-  #fts_enforced = no
+  # Don't fall back to built in search.
+  fts_search_read_fallback = no
 
   # Local filesystem example:
   # Use local filesystem storing FTS indexes
-  fts_dovecot {
+  fts dovecot {
     fs posix {
       prefix = %h/fts/
```

Comment:
doesn't this need dirs setting?

Reply:
I don't think so. I'm not sure why we use it in our CI. Maybe some tests require it. But e.g. CDMI behavior is with "dirs", while S3 behavior is without. So both should work generally.

Reply:
Oh, or was it that we don't want it to autodelete the parent directories.

Reply:
I wonder if we should just rather change the default. Probably better that way.